README.md

```diff
@@ -7,121 +7,147 @@ FlowEHR is a safe, secure &amp; cloud-native development &amp; deployment platfo
 
-2. Create the service principal with required AAD permissions: 
+3. Create the CI resources and service principal with required AAD permissions: 
 
     ```bash
-    make ci-auth
+    make ci
     ```
 
-    _NOTE_: CI deployments require a service principal with access to deploy resources
-    in the subscription, and the following permissions within the associated AAD tenancy:
+    _NOTE_: CI deployments require a service principal with access to deploy resources in the subscription, and the following permissions within the associated AAD tenancy:
     - `Application.ReadWrite.All`: Required to query the directory for the MSGraph app, and create applications used to administer the SQL Server.   
     - `AppRoleAssignment.ReadWrite.All`: Required to assign the following permissions to the System Managed Identity for SQL Server. 
 
-    - Copy the block of JSON from the terminal for the next step.
-
-1. Create and populate a GitHub environment
-
-    Add an environment called `Infra-Test` with the following secrets:
-
-    - `ARM_CLIENT_ID`: Client ID of the service pricipal created in step 2
-    - `ARM_CLIENT_SECRET`: Client secret of the service pricipal created in step 2
-    - `ARM_TENANT_ID`: Tennant ID containing the Azure subscription to deploy into
-    - `ARM_SUBSCRIPTION_ID`: Subscription ID of the Azure subscription to deploy into
-    - `SUFFIX`: Suffix used for naming resources. Must be unique to this repository e.g. `abcd`
-    - `LOCATION`: Name of an Azure location e.g. `uksouth`. These can be listed with `az account list-locations -o table`
-    - `ENVIRONMENT`: Name of the environment e.g. `dev`, also used to name resources
-    - `DEVCONTAINER_ACR_NAME`: Name of the Azure Container Registry to use for the devcontainer build. This may or may not exist. e.g. `flowehrmgmtacr`
-    - `ORG_GH_TOKEN`: GitHub [PAT](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) with scopes to clone any repositories defined in `config.transform.yaml`. This may be added as a repository rather than envrionment secret and be reused betweeen envrionments
-    - `GH_RUNNER_CREATE_TOKEN` Similar to `ORG_GH_TOKEN` but with scopes: "Read access to metadata" and "Read and Write access to administration" on this repository
-    - [Optional] `DATA_SOURCE_CONNECTIONS`: *single line* json containing connectivity information to data sources in the format:
-
-    ```json
-    [
-    {
-        "name": "xxx",
-        "peering": {
-            "virtual_network_name": "xxx",
-            "resource_group_name": "xxx",
-            "dns_zones": [
-                "privatelink.xxx.xxx.azure.com"
-            ]
-        },
-        "fqdn": "<fqdn>",
-        "database_name": "<database_name>",
-        "username": "username",
-        "password": "password"
-    }
-    ]
-    ```
+    - Copy the outputted values to populate in step 5
+
+4. Create GitHub PATs (access tokens)
```

Comment:
I'd suggest we leave this a `TODO` if it's not going to be addressed in this PR, but also 🤷🏼

Reply:
Leave what as TODO? Creating a GitHub App instead? That's tracked as a separate story already

Reply:
suggestion leaving this whole README section as TODO, if we don't actually recommend doing it..

Reply:
Ah ok, well we do as that's the only way it will work at the moment, but we plan on changing it soon yes. I think we leave for now as it's required for the current set up